docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -0,0 +1,162 @@
+ * Terraform
+ * Layer0 Terraform Provider
+
+Layer0 v0.8.4 or later installed and configured. If you have not already configured Layer0, see the [Layer0 installation guide](/setup/install). If you are running an older version of Layer0, see the [Layer0 upgrade instructions](/setup/upgrade#upgrading-older-versions-of-layer0).
+
+See the [Terraform installation guide](/reference/terraform-plugin#install) to install Terraform and the Layer0 Terraform Plugin.
+
+---
+
+## Deploy with Terraform
+Using Terraform, you will deploy a simple guestbook application, which is backed by AWS DynamoDb Table, for persistant storage. The terraform configuration file will use both the Layer0 and AWS Terraform providers, to deploy the guestbook application and provision a new DynamoDb Table.
+
+## Part 1: Clone the Layer0 examples repository
+Run this command to clone the `quintilesims/layer0-examples` repository
+
+<ul>
+  <li class="command">git clone https://github.com/quintilesims/layer0-examples.git</li>
+</ul>
+
+Inside a terminal window, navigate to the `terraform-beyond-layer0 folder`. You should find the following files once you are in the folder. We use these files to set up a Layer0 envrionment and deploy AWS resources with Terraform:
+
+|Filename|Purpose|
+|----|----|
+|[terraform.tfvars](https://github.com/quintilesims/layer0-examples/blob/master/guestbook-db/terraform.tfvars)|Variables specific to the environment and guestbook application|
+|[Dockerrun.aws.json](https://github.com/quintilesims/layer0-examples/blob/master/guestbook-db/Dockerrun.aws.json)|Template for running the guestbook application in a Layer0 environment|
+|[layer0.tf](https://github.com/quintilesims/layer0-examples/blob/master/guestbook-db/layer0.tf)|Provision Layer0 resources|
+
+## Part 2: Terraform Plan
+Before deploying, we can run the follwing command to see what changes Terraform will make to your infrastructure should you go ahead and apply. If you had any errors in your layer0.tf file, running `terraform plan` would output those errors so that you can address them.
+
+<ul>
+  <li class="command">terraform plan</li>
+</ul>
+
+```
++ aws_dynamodb_table.guestbook
+    arn:                       "<computed>"
+    attribute.#:               "1"
+    attribute.4228504427.name: "id"
+    attribute.4228504427.type: "S"
+    hash_key:                  "id"
+    name:                      "guestbook"
+    read_capacity:             "20"
+    stream_arn:                "<computed>"
+    stream_enabled:            "<computed>"
+    stream_view_type:          "<computed>"
+    write_capacity:            "20"
+
+...
+```
+
+## Part 3: Terraform Apply
+Run the follwing command to begin the deploy process. Terraform will prompt you for configuration values that it does not have.
+
+<ul>
+  <li class="command">terraform apply</li>
+</ul>
+
+_To avoid entering these values manually each time you run terraform, you can set the terraform variables by editing the `terraform.tfvars` file._
```

Comment:
We have access to [admonitions](https://pythonhosted.org/Markdown/extensions/admonition.html) in our markdown; I've used one for just this explanation in Walkthrough 1b. It also might be helpful to move this explanation up into the `plan` section, since Terraform will prompt you for these values there as well.
```
!!! Note
	There are a few ways to configure Terraform so that you don't have to keep entering these values every time you run a Terraform command (editing the `terraform.tfvars` file, or exporting evironment variables like `TF_VAR_endpoint` and `TF_VAR_token`, for example). See the [Terraform Docs](https://www.terraform.io/docs/configuration/variables.html) for more.
```